src/mono/wasm/runtime/binding_support.js

```diff
@@ -1461,10 +1480,14 @@ var BindingSupportLib = {
 			// Check to make sure the delegate is still alive on the CLR side of things.
 			if (typeof delegate_obj.__mono_delegate_alive__ !== "undefined") {
 				if (!delegate_obj.__mono_delegate_alive__)
-					throw new Error("The delegate target that is being invoked is no longer available.  Please check if it has been prematurely GC'd.");
+					// HACK: It is possible (though unlikely) for a delegate to be invoked after it's been collected
+					//  if it's being used as a JavaScript event handler and the host environment decides to fire events
+					//  at a point where we've already disposed of the object the event handler is attached to.
+					// As such, we log here instead of throwing an error. We may want to not log at all...
+					console.log("The delegate target that is being invoked is no longer available.  Please check if it has been prematurely GC'd.");
```

Comment:
I think you want to `return` not continue here. I'm not sure what should be return value.